lib/mix/lib/mix/tasks/escript.install.ex

```diff
@@ -23,8 +23,9 @@ defmodule Mix.Tasks.Escript.Install do
       ~/.mix/escripts/foo
 
   For convenience, consider adding `~/.mix/escripts` directory to your
-  `PATH` environment variable. For more information, check the wikipedia
-  article on PATH: https://en.wikipedia.org/wiki/PATH_(variable)
+  `PATH` environment variable. For more information, check the [Wikipedia
+  article on PATH][PATH]
+  [PATH]: https://en.wikipedia.org/wiki/PATH_(variable)
```

Comment:
If we can display the link in a better way, and this is going to be for a release, why display the plain link when we never do that in Elixir documentation?